src/tableView.js

```diff
@@ -364,6 +383,49 @@ class TableView {
     return Math.min(this.instance.columnIndexMapper.getRenderableIndexesLength(), this.settings.maxCols);
   }
 
+  /**
+   * Returns the number of renderable rows.
+   *
+   * @returns {number}
+   */
+  countRenderableRows() {
+    return Math.min(this.instance.rowIndexMapper.getRenderableIndexesLength(), this.settings.maxRows);
+  }
+
+  /**
+   * Returns number of not hidden records counting from the passed starting index.
+   * The counting direction can be controlled by `incrementBy` argument.
+   *
+   * @param {number} visualIndex The visual index from which the counting begins.
+   * @param {string} axix The axis as 'row' or 'column'.
+   * @param {number} incrementBy If `-1` then counting is backwards or forward when `1`.
+   * @returns {number}
+   */
+  countNotHiddenRecords(visualIndex, axix, incrementBy) {
```

Comment:
Coult it be named as `countNotHiddenIndexes`? The current name may suggest that we speak about index paired with something else.